iceoryx_hoofs/source/posix_wrapper/shared_memory_object.cpp

```diff
@@ -112,7 +127,7 @@ SharedMemoryObject::SharedMemoryObject(const SharedMemory::Name_t& name,
                 static_cast<unsigned long long>(memorySizeInBytes),
                 ACCESS_MODE_STRING[static_cast<uint64_t>(accessMode)],
                 OPEN_MODE_STRING[static_cast<uint64_t>(openMode)],
-                baseAddressHint,
+                *baseAddressHint,
```

Comment:
Maybe I missed something, but can we be sure that `baseAddressHint` has a value?

Reply:
good catch!